scripts/install_third_party_test.py

```diff
@@ -375,3 +377,28 @@ def mock_download_and_untar_files(
             with unzip_files_swap, untar_files_swap:
                 install_third_party.main(args=[])
         self.assertEqual(check_function_calls, expected_check_function_calls)
+
+    def test_remove_emojis_from_guppy(self):
```

Comment:
We need to change this test as per the review comments above! The test setup doesn't look good to me here, do you see this pattern anywhere else i.e, making download request for testing?

Reply:
No, I realize that this pattern hasn't been followed anywhere else, but I don't see any other way of testing if the function works correctly.
Should I instead, create a dummy file, append this text to it and then run the function to check if the text got removed or not?
Also, could you please explain why the method that I've implemented is not recommended?

Reply:
Tests should not require connecting to the Internet. It should be possible to run them offline.

Reply:
In that case, @seanlip could you please confirm/deny if the approach I've mentioned above seems reasonable? I don't see any other option. Thanks!

Reply:
We'll have to resolve [this thread](https://github.com/oppia/oppia/pull/9400#discussion_r438941996) first.